cobalt/doc/versioning.md

```diff
@@ -33,46 +37,58 @@ branch is cut, will be modified to start at `1`, and be incremented each time a
-was produced earlier than Cobalt `3.16138`, thus has a lower Build ID.
+Starting with Cobalt 24, the version numbering scheme for **LTS Major and Minor
+Stable releases are now numbered by multiples of 10**. This change aligns with
+semantic versioning, adds more sensible differentiation for Stable vs
+incremental non-Stable releases, and enables the Cobalt team to deliver faster
+fixes, optimizations, and features to our mutual users. **Please only use
+Cobalt LTS releases with `Update Number` that are multiples of 10 for YouTube
+Certification and Cobalt Device Maintenance Updates.**
 
-## Older Cobalt versioning scheme
+NOTE: Pre-built Evergreen binaries are only available for Stable (non-RC)
+releases at this time.
 
-A Cobalt version consists of two components: The Release Number and the Build
-ID. Some real historical Cobalt version examples:
+## For Example:
 
-  * `2.15147`
-  * `3.16138`
-  * `4.16134`
-  * `6.18971`
+|Cobalt Release | Stable (Usable for cert)?|
+|----------|--------|
+|24.lts.1  | No     |
+|24.lts.2  | No     |
+|24.lts.3  | No     |
+|24.lts.10 | **Yes**|
+|24.lts.11 | No     |
+|24.lts.12 | No     |
+|24.lts.20 | **Yes**|
 
-You get the idea. The number before the dot is the "Release Number." The number
-after the dot is the "Build ID."
+## FAQs
 
-### Release Number
+### What version should I use for certification?
 
-In older Cobalt versioning schemes, a "Cobalt release" is an official, tested
-version of Cobalt that is intended to be deployable to production. The
-"Release Number" is a single counting number, starting at "1" for our first
-release, and increasing by one for every release thereafter. This number is
-checked into [`src/cobalt/version.h`](../version.h), and represents **coarse**
-information about the state of the source tree when we decided to do a release.
+- Only Stable Cobalt versions. This is the same policy as before with the
+  Cobalt 24 numbering scheme as the only change.
+- E.g. Cobalt 24.lts.10, 24.lts.20, 24.lts.30,...., 24.lts.50
 
-It is important to note that there are no point releases, or major or minor
-releases. Each release gets its own unique counting number.
 
-## Other Reading
+### What about Evergreen binary versions?
+
+- Stable Evergreen binaries will also be numbered similarly with minor version
+  numbers
+- E.g. 24.lts.10 (Evergreen binary version 4.10.0)
+
+
+### Will Evergreen binaries be provided for all Cobalt releases?
+
+- No, at this time, pre-built Evergreen binaries will only be available for
+  Stable releases (e.g. 24.lts.10, 24.lts.20, …, 24.lts.60). In the future,
+  the Cobalt Team is planning to release matching pre-built Evergreen binaries
+  for RC (non-Stable) releases that will be announced at a later time.
+- Please refer the release page to get all the available binaries:
+  - https://github.com/youtube/cobalt/releases
+
+
+# Other Reading
 
   * [Cobalt Branching](branching.md)
```

Comment:
We will probably want to consider updating this doc with the new language and GitHub details as well.

Reply:
branching.md is updated as well.